src/targets/common.ini

```diff
@@ -73,7 +73,8 @@ monitor_filters = esp8266_exception_decoder
 upload_resetmethod = nodemcu
 bf_upload_command =
 	python "$PROJECT_DIR/python/BFinitPassthrough.py" -b $UPLOAD_SPEED ${UPLOAD_PORT and "-p "+UPLOAD_PORT} -r $PIOENV
-	python "$PROJECT_DIR/python/esptool-3.0/esptool.py" -b $UPLOAD_SPEED ${UPLOAD_PORT and "-p "+UPLOAD_PORT} -c esp8266 --before no_reset --after soft_reset write_flash 0x0000 "$SOURCE"
+	python "$PROJECT_DIR/python/esptool-3.0/esptool.py" -b $UPLOAD_SPEED ${UPLOAD_PORT and "-p "+UPLOAD_PORT} -c esp8266 --before no_reset erase_flash 
```

Comment:
Just gonna tag this line since we can't erase_flash on every upload. Everyone will lose their binding/config.